indexer/packages/postgres/src/db/migrations/migration_files/20220727134954_create_fills.ts

```diff
@@ -19,6 +19,8 @@ export async function up(knex: Knex): Promise<void> {
         'LIMIT',
         'LIQUIDATED',
         'LIQUIDATION',
+        'DELEVERAGED',
+        'OFFSETTING',
```

Comment:
We cannot edit previous migrations in this PR or any future PRs, these will not be noticed by our knex in migrations and will not be included in auxo. Remove this code and add a new knex migrations

Reply:
It's not needed, forgot to revert.